cdap-docs/_common/_source/index.rst

```diff
@@ -14,20 +14,20 @@ CDAP Documentation v\ |version|
 
 .. rubric:: Introduction to the Cask Data Application Platform
 
-The Cask |(TM)| Data Application Platform (CDAP) is an open source application development
+The Cask |(TM)| Data Application Platform (CDAP) is an integrated, open source application development
 platform for the Hadoop ecosystem that provides developers with data and application
-virtualization to accelerate application development, address a broader range of real-time
+abstractions to accelerate application development, address a broader range of real-time
 and batch use cases, and deploy applications into production while satisfying enterprise
 requirements.
 
 CDAP is a layer of software running on top of Apache |(TM)| Hadoop |(R)| platforms such as the
-Cloudera Enterprise Data Hub or the Hortonworks Data Platform. CDAP provides three essential components:
+Cloudera Enterprise Data Hub or the Hortonworks Data Platform. CDAP provides essential capabilities:
```

Comment:
"CDAP provides essential capabilities:" and then following up with a list of things sounds awkward to me. Maybe "CDAP provides N essential capabilities", as before, or "CDAP provides various essential capabilities" or "CDAP provides the following capabilities"